flepimop/gempyor_pkg/src/gempyor/inference.py

```diff
@@ -164,7 +164,7 @@ def get_static_arguments(modinf: model_info.ModelInfo):
     )
 
     initial_conditions = modinf.initial_conditions.get_from_config(sim_id=0, modinf=modinf)
-    seeding_data, seeding_amounts = modinf.seeding.get_from_config(sim_id=0, modinf=modinf)
+    seeding_data, seeding_amounts = modinf.get_seeding_data(0)
```

Comment:
seems preferrable to leave the explicity `sim_id=0` style

Reply:
Not sure I see the value since `sim_id` is the only argument, but done.